base/docker-compose.yml

```
@@ -1,42 +1,47 @@
 services:
```

This will change the default Docker compose project name from "base" (folder name) to "bleemeo-community-base"

```suggestion
name: bleemeo-community-edition-base

services:
```